### PR TITLE
Resolves: Undefined Reference to GetTensorView

### PR DESCRIPTION
**Pull request by imriss**
This resolves the error mentioned in: https://github.com/Microsoft/CNTK/issues/2025

### CODE REVIEW

**Comment by zhouwangzw** [comment]
@wolfma is added to the review. #Closed

**Comment by zhouwangzw** [comment]
@eldak is added to the review. #Closed

**Comment by zhouwangzw** [comment]
This is the pull request 2104 to fix compilation error under GCC 7.1.

---
In reply to: [314147365](https://github.com/Microsoft/CNTK/pull/2104#issuecomment-314147365) [](ancestors = 314147365)

File: Source/CNTKv2LibraryDll/NDArrayView.cpp [hunk 1]
```diff
@@ -577,6 +577,9 @@ namespace CNTK
 
     template CNTK_API const float* NDArrayView::DataBuffer<float>() const;
     template CNTK_API const double* NDArrayView::DataBuffer<double>() const;
+
```
Comment (zhouwangzw) [inline Source/CNTKv2LibraryDll/NDArrayView.cpp]: Could you please remove the white spaces?

Comment (imriss) [inline Source/CNTKv2LibraryDll/NDArrayView.cpp]: Done

**Comment by wolfma61** [comment]
:shipit:

**Pull request merged**